tests/test_server.py

```diff
@@ -1421,7 +1363,10 @@ async def test_exception_response():
 
     response = await client.post(
         "/query",
-        json={"message": {"query_graph": qgraph}},
+        json={
+            "message": {"query_graph": qgraph},
+            "log_level": "DEBUG",
```

Comment:
`log_level` can be removed in all tests.

Reply:
64f0117eee0f1eaa6db46d8cec0461ac12788d6f